<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="0"
            tests="0">

  <testsuite name="tests_Joseph"
    errors="0" failures="0" hostname="" id="" package="tests_Joseph" skipped="0"
    tests="0" time="0.000" timestamp="Wed, 03 Jan 2018 17:10:19 GMT">
  

  
    <system-err>
        Error while running [Nathan Tests / I can delete an employee and verify their name is no longer in the employee list]:

  TypeError: Cannot read property 'to' of undefined
    at Object.I can delete an employee and verify their name is no longer in the employee list (C:\Users\nedwi\OneDrive\Documents\Devmountain\Code\employee-manager-v2\nightwatch\tests\nathan_tests.js:28:37)
    at Module.call (C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:62:34)
    at C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:70:29
    at _fulfilled (C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\nedwi\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    </system-err>
  

  
  </testsuite>
</testsuites>
